support/ruby/collins-client/Gemfile.lock

```diff
@@ -1,50 +0,0 @@
-GEM
```

Comment:
If we want exact versions of gems we depend on I think those should go in the gemspec. I can add some pessimistic version requirements to the gemspec based on what I've been using when testing it on the various versions of ruby.